lib/Helper/Requirements.php

```diff
@@ -58,7 +58,7 @@ public function getFaceDetectionModel() {
 	private function getModel1File(string $file) {
 		$fullPath = $this->modelService->getModelPath($file);
 
-		if (file_exists($fullPath)) {
+		if (true || file_exists($fullPath)) {
```

Comment:
Bypass to always return fullpath.. Otherwise, when you try to open the model, the file will be null.